log4j-api/src/main/java/org/apache/logging/log4j/util/LowLevelLogUtil.java

```diff
@@ -32,6 +32,7 @@
  */
 final class LowLevelLogUtil {
 
+    @SuppressWarnings("DefaultCharset")
```

Comment:
Might be useful to leave a comment, too, that this is intentional functionality? Though I suppose the warning suppression sort of implies that.

Reply:
Sure, I'll add some comments and links to the error descriptions.